.gitignore

```diff
@@ -2,3 +2,5 @@
 .nyc_output/
 node_modules/
 npm-debug.log
+/.idea/
```

Comment:
Nitpick: Stuff like that should be in your ~/.gitignore, not in the .gitignore of each project. But whatever :-)

Reply:
I did not know that git also looks into .gitignore files in the home directory. Neat. I removed this.